micrometer-core/src/main/java/io/micrometer/core/instrument/observation/TimerObservationHandler.java

```diff
@@ -15,10 +15,7 @@
  */
 package io.micrometer.core.instrument.observation;
 
-import io.micrometer.core.instrument.MeterRegistry;
-import io.micrometer.core.instrument.Tag;
-import io.micrometer.core.instrument.Tags;
-import io.micrometer.core.instrument.Timer;
+import io.micrometer.core.instrument.*;
```

Comment:
Do we allow for wildcard imports?   Annoying, aren't I? :-D

Reply:
Yes (I mean the wildcard imports :D), IntelliJ automatically does this, based on our [`.editorconfig`](https://github.com/micrometer-metrics/micrometer/blob/main/.editorconfig) file